### PR TITLE
black=21.5b2

### PR DESCRIPTION
**Pull request by trexfeathers**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by trexfeathers** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "21.5b2" %}
 
 package:
   name: {{ name|lower }}-recipe
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 23695358dbcb3deafe7f0a3ad89feee5999a46be5fec21f4f1d108be0bcdb3b1
+  sha256: a85cfc9d2295c0ae7172ad8b49823abb11627709ba1bd661ffb00f184da85280
```
Comment (carlodri) [inline recipe/meta.yaml:10]:
```suggestion
  sha256: 1fc0e0a2c8ae7d269dfcf0c60a89afa299664f3e811395d40b1922dff8f854b5
```

Comment (trexfeathers) [reply recipe/meta.yaml:10]: Yes, I see, thanks - I should have got the SHA from the PyPI file instead of the GitHub file!

**Comment by carlodri** [comment]
thanks @trexfeathers!

**Pull request merged**